suites/tentacle/nfs/tier0-nfs-ganesha.yaml

```diff
@@ -343,22 +371,6 @@ tests:
         qos_type: PerClient
         operation: restart
 
-  - test:
```

Comment:
Removing from this suite is fine. however, the tests are not in any other suite so we would need to add a suite for qos to run in regression pipeline so that we dont miss coverage because of removal of tests.